jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -422,8 +425,10 @@ def __init__(self, client: Client, **kwargs):
         self._datum_index = 0
         self._mount_time = datetime.datetime.min
         self._input = None
+        self._download_dir = None
         shutil.rmtree(f"{self._FILEINFO_DIR}", ignore_errors=True)
         os.makedirs(self._FILEINFO_DIR, exist_ok=True)
+        os.makedirs(PFS_MOUNT_DIR, exist_ok=True)
```

Comment:
We might want to think about the error state here. While it may not be something we care about, if you are running this locally on a mac it will try to create the /pfs directory which mac won't let you do.

Reply:
I was thinking that if this call fails then the extension will fail to start, which I think is acceptable. I suppose it could be more clear if we log a message saying to set the `PFS_MOUNT_DIR` env var to something that would work on their system.